app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/order/Order.java

```diff
@@ -0,0 +1,113 @@
+package gov.hhs.cdc.trustedintermediary.etor.order;
+
+import gov.hhs.cdc.trustedintermediary.context.ApplicationContext;
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Order {
+    private String id;
+    private String destination;
+    private String createAt;
+    private String client;
+    private String body;
+    private OrderMessage orderMessage = new OrderMessage();
```

Comment:
I don't fully understand why the `Order` has an `OrderMessage` field.  Can't the `Order` be independent?  Then, to construct an `OrderMessage`, we just have a constructor that takes an `Order`.  E.g.

```java
Order order = new Order(/*<interesting arguments>*/);
OrderMessage orderMessage = new OrderMessage(order);
```

And then the `OrderMessage` constructor contains some of the logic from the `Order#generateMessage` method.

I think there is also something to be said for making the `OrderMessage` simpler so it isn't just duplicating a lot of the fields here in `Order`.

Reply:
I like that. Added OrderMessage contructor that takes in an order. The think the OrderMessage will eventually change to return a more customed message. But, I don't think we can get rid of the main fields like id, destination and createdAt. In fact, I don't think we need anything else besided those three fields. Maybe we don't need the OrderMessage class. We can probably construct the message with a helper function inside the OrderController. Thought?

Reply:
I'm totally cool reducing the number of fields in `OrderMessage`.  I'm also OK keeping `OrderMessage` even if there are only three fields, but that can be up to you.